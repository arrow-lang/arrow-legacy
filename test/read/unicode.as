遠く離れ、ワード山の背後に、遠い国からVokaliaと
Consonantiaは、盲目のテキストがそこに住みます。分離された彼らはBookmarksgroveに住んでいます
右の意味、大きな言語海の海岸で。という小さな川
デューデンは、その場所によって流れ、必要なregelialiaでそれを供給しています。それは
文章のローストの部分は、あなたに飛ぶれるparadisematic国、
口。でもすべての強力なポインティングはそれが盲目の文章について管理していません
しかしほとんどunorthographic生活1日ブラインドテキストの小さなライン
Loremのイプサムの名前は文法の遠い世界に向けて出発することにしました。ビッグOxmox
悪いカンマの数千が、野生あったので、彼女がそうしないことをお勧め
疑問符とよこしまSemikoliが、少しブラインドテキスト聞​​いていませんでした。
彼女は、彼女の7 versaliaを詰めベルトに彼女の最初のを入れて、自分自身を上に作られました
方法。彼女はイタリック山脈の最初の丘陵に達したとき、彼女が持っていました
バック彼女の故郷Bookmarksgrove、の見出しのスカイラインの最後のビュー
アルファベット村と彼女自身の道のサブライン、ラインレーン。
