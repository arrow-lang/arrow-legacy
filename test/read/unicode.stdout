 1.1         00 00 90 60  遠
 1.2         00 00 30 4f  く
 1.3         00 00 96 e2  離
 1.4         00 00 30 8c  れ
 1.5         00 00 30 01  、
 1.6         00 00 30 ef  ワ
 1.7         00 00 30 fc  ー
 1.8         00 00 30 c9  ド
 1.9         00 00 5c 71  山
 1.10        00 00 30 6e  の
 1.11        00 00 80 cc  背
 1.12        00 00 5f 8c  後
 1.13        00 00 30 6b  に
 1.14        00 00 30 01  、
 1.15        00 00 90 60  遠
 1.16        00 00 30 44  い
 1.17        00 00 56 fd  国
 1.18        00 00 30 4b  か
 1.19        00 00 30 89  ら
 1.20        00 00 00 56  V
 1.21        00 00 00 6f  o
 1.22        00 00 00 6b  k
 1.23        00 00 00 61  a
 1.24        00 00 00 6c  l
 1.25        00 00 00 69  i
 1.26        00 00 00 61  a
 1.27        00 00 30 68  と
 1.28        00 00 00 0a  
 2.1         00 00 00 43  C
 2.2         00 00 00 6f  o
 2.3         00 00 00 6e  n
 2.4         00 00 00 73  s
 2.5         00 00 00 6f  o
 2.6         00 00 00 6e  n
 2.7         00 00 00 61  a
 2.8         00 00 00 6e  n
 2.9         00 00 00 74  t
 2.10        00 00 00 69  i
 2.11        00 00 00 61  a
 2.12        00 00 30 6f  は
 2.13        00 00 30 01  、
 2.14        00 00 76 f2  盲
 2.15        00 00 76 ee  目
 2.16        00 00 30 6e  の
 2.17        00 00 30 c6  テ
 2.18        00 00 30 ad  キ
 2.19        00 00 30 b9  ス
 2.20        00 00 30 c8  ト
 2.21        00 00 30 4c  が
 2.22        00 00 30 5d  そ
 2.23        00 00 30 53  こ
 2.24        00 00 30 6b  に
 2.25        00 00 4f 4f  住
 2.26        00 00 30 7f  み
 2.27        00 00 30 7e  ま
 2.28        00 00 30 59  す
 2.29        00 00 30 02  。
 2.30        00 00 52 06  分
 2.31        00 00 96 e2  離
 2.32        00 00 30 55  さ
 2.33        00 00 30 8c  れ
 2.34        00 00 30 5f  た
 2.35        00 00 5f 7c  彼
 2.36        00 00 30 89  ら
 2.37        00 00 30 6f  は
 2.38        00 00 00 42  B
 2.39        00 00 00 6f  o
 2.40        00 00 00 6f  o
 2.41        00 00 00 6b  k
 2.42        00 00 00 6d  m
 2.43        00 00 00 61  a
 2.44        00 00 00 72  r
 2.45        00 00 00 6b  k
 2.46        00 00 00 73  s
 2.47        00 00 00 67  g
 2.48        00 00 00 72  r
 2.49        00 00 00 6f  o
 2.50        00 00 00 76  v
 2.51        00 00 00 65  e
 2.52        00 00 30 6b  に
 2.53        00 00 4f 4f  住
 2.54        00 00 30 93  ん
 2.55        00 00 30 67  で
 2.56        00 00 30 44  い
 2.57        00 00 30 7e  ま
 2.58        00 00 30 59  す
 2.59        00 00 00 0a  
 3.1         00 00 53 f3  右
 3.2         00 00 30 6e  の
 3.3         00 00 61 0f  意
 3.4         00 00 54 73  味
 3.5         00 00 30 01  、
 3.6         00 00 59 27  大
 3.7         00 00 30 4d  き
 3.8         00 00 30 6a  な
 3.9         00 00 8a 00  言
 3.10        00 00 8a 9e  語
 3.11        00 00 6d 77  海
 3.12        00 00 30 6e  の
 3.13        00 00 6d 77  海
 3.14        00 00 5c b8  岸
 3.15        00 00 30 67  で
 3.16        00 00 30 02  。
 3.17        00 00 30 68  と
 3.18        00 00 30 44  い
 3.19        00 00 30 46  う
 3.20        00 00 5c 0f  小
 3.21        00 00 30 55  さ
 3.22        00 00 30 6a  な
 3.23        00 00 5d dd  川
 3.24        00 00 00 0a  
 4.1         00 00 30 c7  デ
 4.2         00 00 30 e5  ュ
 4.3         00 00 30 fc  ー
 4.4         00 00 30 c7  デ
 4.5         00 00 30 f3  ン
 4.6         00 00 30 6f  は
 4.7         00 00 30 01  、
 4.8         00 00 30 5d  そ
 4.9         00 00 30 6e  の
 4.10        00 00 58 34  場
 4.11        00 00 62 40  所
 4.12        00 00 30 6b  に
 4.13        00 00 30 88  よ
 4.14        00 00 30 63  っ
 4.15        00 00 30 66  て
 4.16        00 00 6d 41  流
 4.17        00 00 30 8c  れ
 4.18        00 00 30 01  、
 4.19        00 00 5f c5  必
 4.20        00 00 89 81  要
 4.21        00 00 30 6a  な
 4.22        00 00 00 72  r
 4.23        00 00 00 65  e
 4.24        00 00 00 67  g
 4.25        00 00 00 65  e
 4.26        00 00 00 6c  l
 4.27        00 00 00 69  i
 4.28        00 00 00 61  a
 4.29        00 00 00 6c  l
 4.30        00 00 00 69  i
 4.31        00 00 00 61  a
 4.32        00 00 30 67  で
 4.33        00 00 30 5d  そ
 4.34        00 00 30 8c  れ
 4.35        00 00 30 92  を
 4.36        00 00 4f 9b  供
 4.37        00 00 7d 66  給
 4.38        00 00 30 57  し
 4.39        00 00 30 66  て
 4.40        00 00 30 44  い
 4.41        00 00 30 7e  ま
 4.42        00 00 30 59  す
 4.43        00 00 30 02  。
 4.44        00 00 30 5d  そ
 4.45        00 00 30 8c  れ
 4.46        00 00 30 6f  は
 4.47        00 00 00 0a  
 5.1         00 00 65 87  文
 5.2         00 00 7a e0  章
 5.3         00 00 30 6e  の
 5.4         00 00 30 ed  ロ
 5.5         00 00 30 fc  ー
 5.6         00 00 30 b9  ス
 5.7         00 00 30 c8  ト
 5.8         00 00 30 6e  の
 5.9         00 00 90 e8  部
 5.10        00 00 52 06  分
 5.11        00 00 30 6f  は
 5.12        00 00 30 01  、
 5.13        00 00 30 42  あ
 5.14        00 00 30 6a  な
 5.15        00 00 30 5f  た
 5.16        00 00 30 6b  に
 5.17        00 00 98 db  飛
 5.18        00 00 30 76  ぶ
 5.19        00 00 30 8c  れ
 5.20        00 00 30 8b  る
 5.21        00 00 00 70  p
 5.22        00 00 00 61  a
 5.23        00 00 00 72  r
 5.24        00 00 00 61  a
 5.25        00 00 00 64  d
 5.26        00 00 00 69  i
 5.27        00 00 00 73  s
 5.28        00 00 00 65  e
 5.29        00 00 00 6d  m
 5.30        00 00 00 61  a
 5.31        00 00 00 74  t
 5.32        00 00 00 69  i
 5.33        00 00 00 63  c
 5.34        00 00 56 fd  国
 5.35        00 00 30 01  、
 5.36        00 00 00 0a  
 6.1         00 00 53 e3  口
 6.2         00 00 30 02  。
 6.3         00 00 30 67  で
 6.4         00 00 30 82  も
 6.5         00 00 30 59  す
 6.6         00 00 30 79  べ
 6.7         00 00 30 66  て
 6.8         00 00 30 6e  の
 6.9         00 00 5f 37  強
 6.10        00 00 52 9b  力
 6.11        00 00 30 6a  な
 6.12        00 00 30 dd  ポ
 6.13        00 00 30 a4  イ
 6.14        00 00 30 f3  ン
 6.15        00 00 30 c6  テ
 6.16        00 00 30 a3  ィ
 6.17        00 00 30 f3  ン
 6.18        00 00 30 b0  グ
 6.19        00 00 30 6f  は
 6.20        00 00 30 5d  そ
 6.21        00 00 30 8c  れ
 6.22        00 00 30 4c  が
 6.23        00 00 76 f2  盲
 6.24        00 00 76 ee  目
 6.25        00 00 30 6e  の
 6.26        00 00 65 87  文
 6.27        00 00 7a e0  章
 6.28        00 00 30 6b  に
 6.29        00 00 30 64  つ
 6.30        00 00 30 44  い
 6.31        00 00 30 66  て
 6.32        00 00 7b a1  管
 6.33        00 00 74 06  理
 6.34        00 00 30 57  し
 6.35        00 00 30 66  て
 6.36        00 00 30 44  い
 6.37        00 00 30 7e  ま
 6.38        00 00 30 5b  せ
 6.39        00 00 30 93  ん
 6.40        00 00 00 0a  
 7.1         00 00 30 57  し
 7.2         00 00 30 4b  か
 7.3         00 00 30 57  し
 7.4         00 00 30 7b  ほ
 7.5         00 00 30 68  と
 7.6         00 00 30 93  ん
 7.7         00 00 30 69  ど
 7.8         00 00 00 75  u
 7.9         00 00 00 6e  n
 7.10        00 00 00 6f  o
 7.11        00 00 00 72  r
 7.12        00 00 00 74  t
 7.13        00 00 00 68  h
 7.14        00 00 00 6f  o
 7.15        00 00 00 67  g
 7.16        00 00 00 72  r
 7.17        00 00 00 61  a
 7.18        00 00 00 70  p
 7.19        00 00 00 68  h
 7.20        00 00 00 69  i
 7.21        00 00 00 63  c
 7.22        00 00 75 1f  生
 7.23        00 00 6d 3b  活
 7.24        00 00 00 31  1
 7.25        00 00 65 e5  日
 7.26        00 00 30 d6  ブ
 7.27        00 00 30 e9  ラ
 7.28        00 00 30 a4  イ
 7.29        00 00 30 f3  ン
 7.30        00 00 30 c9  ド
 7.31        00 00 30 c6  テ
 7.32        00 00 30 ad  キ
 7.33        00 00 30 b9  ス
 7.34        00 00 30 c8  ト
 7.35        00 00 30 6e  の
 7.36        00 00 5c 0f  小
 7.37        00 00 30 55  さ
 7.38        00 00 30 6a  な
 7.39        00 00 30 e9  ラ
 7.40        00 00 30 a4  イ
 7.41        00 00 30 f3  ン
 7.42        00 00 00 0a  
 8.1         00 00 00 4c  L
 8.2         00 00 00 6f  o
 8.3         00 00 00 72  r
 8.4         00 00 00 65  e
 8.5         00 00 00 6d  m
 8.6         00 00 30 6e  の
 8.7         00 00 30 a4  イ
 8.8         00 00 30 d7  プ
 8.9         00 00 30 b5  サ
 8.10        00 00 30 e0  ム
 8.11        00 00 30 6e  の
 8.12        00 00 54 0d  名
 8.13        00 00 52 4d  前
 8.14        00 00 30 6f  は
 8.15        00 00 65 87  文
 8.16        00 00 6c d5  法
 8.17        00 00 30 6e  の
 8.18        00 00 90 60  遠
 8.19        00 00 30 44  い
 8.20        00 00 4e 16  世
 8.21        00 00 75 4c  界
 8.22        00 00 30 6b  に
 8.23        00 00 54 11  向
 8.24        00 00 30 51  け
 8.25        00 00 30 66  て
 8.26        00 00 51 fa  出
 8.27        00 00 76 7a  発
 8.28        00 00 30 59  す
 8.29        00 00 30 8b  る
 8.30        00 00 30 53  こ
 8.31        00 00 30 68  と
 8.32        00 00 30 6b  に
 8.33        00 00 30 57  し
 8.34        00 00 30 7e  ま
 8.35        00 00 30 57  し
 8.36        00 00 30 5f  た
 8.37        00 00 30 02  。
 8.38        00 00 30 d3  ビ
 8.39        00 00 30 c3  ッ
 8.40        00 00 30 b0  グ
 8.41        00 00 00 4f  O
 8.42        00 00 00 78  x
 8.43        00 00 00 6d  m
 8.44        00 00 00 6f  o
 8.45        00 00 00 78  x
 8.46        00 00 00 0a  
 9.1         00 00 60 aa  悪
 9.2         00 00 30 44  い
 9.3         00 00 30 ab  カ
 9.4         00 00 30 f3  ン
 9.5         00 00 30 de  マ
 9.6         00 00 30 6e  の
 9.7         00 00 65 70  数
 9.8         00 00 53 43  千
 9.9         00 00 30 4c  が
 9.10        00 00 30 01  、
 9.11        00 00 91 ce  野
 9.12        00 00 75 1f  生
 9.13        00 00 30 42  あ
 9.14        00 00 30 63  っ
 9.15        00 00 30 5f  た
 9.16        00 00 30 6e  の
 9.17        00 00 30 67  で
 9.18        00 00 30 01  、
 9.19        00 00 5f 7c  彼
 9.20        00 00 59 73  女
 9.21        00 00 30 4c  が
 9.22        00 00 30 5d  そ
 9.23        00 00 30 46  う
 9.24        00 00 30 57  し
 9.25        00 00 30 6a  な
 9.26        00 00 30 44  い
 9.27        00 00 30 53  こ
 9.28        00 00 30 68  と
 9.29        00 00 30 92  を
 9.30        00 00 30 4a  お
 9.31        00 00 52 e7  勧
 9.32        00 00 30 81  め
 9.33        00 00 00 0a  
 10.1        00 00 75 91  疑
 10.2        00 00 55 4f  問
 10.3        00 00 7b 26  符
 10.4        00 00 30 68  と
 10.5        00 00 30 88  よ
 10.6        00 00 30 53  こ
 10.7        00 00 30 57  し
 10.8        00 00 30 7e  ま
 10.9        00 00 00 53  S
 10.10       00 00 00 65  e
 10.11       00 00 00 6d  m
 10.12       00 00 00 69  i
 10.13       00 00 00 6b  k
 10.14       00 00 00 6f  o
 10.15       00 00 00 6c  l
 10.16       00 00 00 69  i
 10.17       00 00 30 4c  が
 10.18       00 00 30 01  、
 10.19       00 00 5c 11  少
 10.20       00 00 30 57  し
 10.21       00 00 30 d6  ブ
 10.22       00 00 30 e9  ラ
 10.23       00 00 30 a4  イ
 10.24       00 00 30 f3  ン
 10.25       00 00 30 c9  ド
 10.26       00 00 30 c6  テ
 10.27       00 00 30 ad  キ
 10.28       00 00 30 b9  ス
 10.29       00 00 30 c8  ト
 10.30       00 00 80 5e  聞
 10.31       00 00 20 0b  ​
 10.32       00 00 20 0b  ​
 10.33       00 00 30 44  い
 10.34       00 00 30 66  て
 10.35       00 00 30 44  い
 10.36       00 00 30 7e  ま
 10.37       00 00 30 5b  せ
 10.38       00 00 30 93  ん
 10.39       00 00 30 67  で
 10.40       00 00 30 57  し
 10.41       00 00 30 5f  た
 10.42       00 00 30 02  。
 10.43       00 00 00 0a  
 11.1        00 00 5f 7c  彼
 11.2        00 00 59 73  女
 11.3        00 00 30 6f  は
 11.4        00 00 30 01  、
 11.5        00 00 5f 7c  彼
 11.6        00 00 59 73  女
 11.7        00 00 30 6e  の
 11.8        00 00 00 37  7
 11.9        00 00 00 20   
 11.10       00 00 00 76  v
 11.11       00 00 00 65  e
 11.12       00 00 00 72  r
 11.13       00 00 00 73  s
 11.14       00 00 00 61  a
 11.15       00 00 00 6c  l
 11.16       00 00 00 69  i
 11.17       00 00 00 61  a
 11.18       00 00 30 92  を
 11.19       00 00 8a 70  詰
 11.20       00 00 30 81  め
 11.21       00 00 30 d9  ベ
 11.22       00 00 30 eb  ル
 11.23       00 00 30 c8  ト
 11.24       00 00 30 6b  に
 11.25       00 00 5f 7c  彼
 11.26       00 00 59 73  女
 11.27       00 00 30 6e  の
 11.28       00 00 67 00  最
 11.29       00 00 52 1d  初
 11.30       00 00 30 6e  の
 11.31       00 00 30 92  を
 11.32       00 00 51 65  入
 11.33       00 00 30 8c  れ
 11.34       00 00 30 66  て
 11.35       00 00 30 01  、
 11.36       00 00 81 ea  自
 11.37       00 00 52 06  分
 11.38       00 00 81 ea  自
 11.39       00 00 8e ab  身
 11.40       00 00 30 92  を
 11.41       00 00 4e 0a  上
 11.42       00 00 30 6b  に
 11.43       00 00 4f 5c  作
 11.44       00 00 30 89  ら
 11.45       00 00 30 8c  れ
 11.46       00 00 30 7e  ま
 11.47       00 00 30 57  し
 11.48       00 00 30 5f  た
 11.49       00 00 00 0a  
 12.1        00 00 65 b9  方
 12.2        00 00 6c d5  法
 12.3        00 00 30 02  。
 12.4        00 00 5f 7c  彼
 12.5        00 00 59 73  女
 12.6        00 00 30 6f  は
 12.7        00 00 30 a4  イ
 12.8        00 00 30 bf  タ
 12.9        00 00 30 ea  リ
 12.10       00 00 30 c3  ッ
 12.11       00 00 30 af  ク
 12.12       00 00 5c 71  山
 12.13       00 00 81 08  脈
 12.14       00 00 30 6e  の
 12.15       00 00 67 00  最
 12.16       00 00 52 1d  初
 12.17       00 00 30 6e  の
 12.18       00 00 4e 18  丘
 12.19       00 00 96 75  陵
 12.20       00 00 30 6b  に
 12.21       00 00 90 54  達
 12.22       00 00 30 57  し
 12.23       00 00 30 5f  た
 12.24       00 00 30 68  と
 12.25       00 00 30 4d  き
 12.26       00 00 30 01  、
 12.27       00 00 5f 7c  彼
 12.28       00 00 59 73  女
 12.29       00 00 30 4c  が
 12.30       00 00 63 01  持
 12.31       00 00 30 63  っ
 12.32       00 00 30 66  て
 12.33       00 00 30 44  い
 12.34       00 00 30 7e  ま
 12.35       00 00 30 57  し
 12.36       00 00 30 5f  た
 12.37       00 00 00 0a  
 13.1        00 00 30 d0  バ
 13.2        00 00 30 c3  ッ
 13.3        00 00 30 af  ク
 13.4        00 00 5f 7c  彼
 13.5        00 00 59 73  女
 13.6        00 00 30 6e  の
 13.7        00 00 65 45  故
 13.8        00 00 90 f7  郷
 13.9        00 00 00 42  B
 13.10       00 00 00 6f  o
 13.11       00 00 00 6f  o
 13.12       00 00 00 6b  k
 13.13       00 00 00 6d  m
 13.14       00 00 00 61  a
 13.15       00 00 00 72  r
 13.16       00 00 00 6b  k
 13.17       00 00 00 73  s
 13.18       00 00 00 67  g
 13.19       00 00 00 72  r
 13.20       00 00 00 6f  o
 13.21       00 00 00 76  v
 13.22       00 00 00 65  e
 13.23       00 00 30 01  、
 13.24       00 00 30 6e  の
 13.25       00 00 89 8b  見
 13.26       00 00 51 fa  出
 13.27       00 00 30 57  し
 13.28       00 00 30 6e  の
 13.29       00 00 30 b9  ス
 13.30       00 00 30 ab  カ
 13.31       00 00 30 a4  イ
 13.32       00 00 30 e9  ラ
 13.33       00 00 30 a4  イ
 13.34       00 00 30 f3  ン
 13.35       00 00 30 6e  の
 13.36       00 00 67 00  最
 13.37       00 00 5f 8c  後
 13.38       00 00 30 6e  の
 13.39       00 00 30 d3  ビ
 13.40       00 00 30 e5  ュ
 13.41       00 00 30 fc  ー
 13.42       00 00 00 0a  
 14.1        00 00 30 a2  ア
 14.2        00 00 30 eb  ル
 14.3        00 00 30 d5  フ
 14.4        00 00 30 a1  ァ
 14.5        00 00 30 d9  ベ
 14.6        00 00 30 c3  ッ
 14.7        00 00 30 c8  ト
 14.8        00 00 67 51  村
 14.9        00 00 30 68  と
 14.10       00 00 5f 7c  彼
 14.11       00 00 59 73  女
 14.12       00 00 81 ea  自
 14.13       00 00 8e ab  身
 14.14       00 00 30 6e  の
 14.15       00 00 90 53  道
 14.16       00 00 30 6e  の
 14.17       00 00 30 b5  サ
 14.18       00 00 30 d6  ブ
 14.19       00 00 30 e9  ラ
 14.20       00 00 30 a4  イ
 14.21       00 00 30 f3  ン
 14.22       00 00 30 01  、
 14.23       00 00 30 e9  ラ
 14.24       00 00 30 a4  イ
 14.25       00 00 30 f3  ン
 14.26       00 00 30 ec  レ
 14.27       00 00 30 fc  ー
 14.28       00 00 30 f3  ン
 14.29       00 00 30 02  。
 14.30       00 00 00 0a  
